etc/parse_libs.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+## create json for all files
+
+/home/sebastian/Software/Singular-git/Singular/libparse -f $1 | awk -F " " '{print "[\"" $1 "\",\"" $3 "\"]\,"}' | tail -n+4
```

Comment:
That seems rather specific to your system ;-)

Reply:
Indeed. Work in progress atm ;)